Core/sia380/sia380.py

```diff
@@ -0,0 +1,348 @@
+
+    tokWh = 1000.0
+    return [x / tokWh for x in Q_Heat], [x / tokWh for x in Q_Cool], [x / tokWh for x in Q_Elec], Q_T, Q_V, Q_i, Q_s
+
+
+def read_building_json(room, dayspermonth):
+    """
+
+    :param room: Room description as json
+    :return: room properties
+    """
+
+    """
+        tau     Zeitkonstante des Gebäudes [h]
+        theta_e Aussenlufttemperatur
+        theta_i Raumlufttemperatur
+        t       Länge der Berechnungsperiode [h]
+        A_th    Thermische Gebäudehüllfläche [m2] 
+        A_w     Fensterfläche [m2]                      !!!!!! f_g in sia2024 - Glasanteil in [%]
+        U_op    Wärmedurchgangskoeffizient Aussenwand [W/m2K]
+        U_w     Wärmedurchgangskoeffizient Fenster [W/m2K]
+        Vdot_e_spec  Aussenluft-Volumenstrom [m3/m2h]
+        Vdot_inf_spec Aussenluft-Volumenstrom durch Infiltration [m3/m2h]
+        eta_rec Nutzungsgrad der Wärmerückgewinnung [-]
+        phi_P   Wärmeabgabe Personen [W/m2]
+        phi_L   Wärmeabgabe Beleuchtung [W/m2]
+        phi_A   Wärmeabgabe Geräte [W/m2]
+        t_P     Vollaststunden Personen [h]
+        t_L     Vollaststunden Beleuchtung [h]
+        t_A     Vollaststunden Geräte [h]
+        g       g-Wert [-]
+        f_sh    Reduktionsfaktor solare Wärmeeinträge [-]
+        I       description": "Solare Strahlung [Wh/m2]
+    """
+
+    """
+    cases according to SIA 2024:2015.
+    The norm defines following ROOM and/or BUILDING types (p.7-8):
+        _____________________________________________________________________________________
+              | abbr.     | description                                    |    code SIA 380
+        ______|___________|________________________________________________|_________________
+        - 1.1   mfh:        multi-family house (Mehrfamilienhaus)                   HNF1
+        - 1.2   efh:        single family house (Einfamilienhaus)                   HNF1
+        - 2.1   ...:        Hotelroom                                               HNF1
+        - 2.2   ...:        Hotel lobby                                             HNF1
+        - ...   ...:
+        - 3.1   office:     small office space (Einzel,- Gruppenbüro)               HNF2
+        - ...   ...:
+        - 4.1   school:     school room (Schulzimmer)                               HNF5
+        - ...   ...:
+        _____________________________________________________________________________________
+    """
+
+    f_sh = 0.9  # sia2024, p.12, 1.3.1.9 Reduktion solare Wärmeeinträge
+
+    tau = room["Zeitkonstante"]
+    theta_i_summer = room["Raumlufttemperatur Auslegung Kuehlung (Sommer)"]
+    theta_i_winter = room["Raumlufttemperatur Auslegung Heizen (Winter)"]
+
+    U_op = room["U-Wert opake Bauteile"]
```

Comment:
this is going to get ugly when we translate... where does `room` come from again? why can't it use `room["tau"]`? (keep the translation somewhere in a dictionary / lookup table if you need it again.

Reply:
ok yes I agree. I'll keep it nevertheless for this PR, because issue #281 will introduce some changes here. It will make sia380.py read building elements (containing individual construction properties), instead of the room json.

Reply:
oh it actually could use tau = room["Zeitkonstante"]...

room is the sia2024 json that contains room properties

Reply:
regarding the lookup table.. I don't know where to put it, so for now I just dumped it as doc string at the top of the script